src/lib_wasm.rs

```diff
@@ -161,17 +165,58 @@ impl FileReader<Cursor<Vec<u8>>> for WasmFileReader {
     }
 }
 
+fn generate_prefix_stmts(config: &Config) -> Vec<Stmt> {
```

Comment:
i'd move this method to `rewriter.rs`
`lib_wasm.rs` is the 'public' interface for wasm as `lib_napi.rs` is the interface for napi so I try to keep `lib_wasm.rs` swc agnostic (as you can see it didn't import swc crates before)

I'm preparing a PR with the suggestion :)